Charon/VirtualFile.py

```diff
@@ -1,23 +1,26 @@
 # Copyright (c) 2018 Ultimaker B.V.
 # libCharon is released under the terms of the LGPLv3 or higher.
+import os
 
-import os.path #To get the file extension from a path.
+from Charon.FileInterface import FileInterface  # The interface we're implementing.
+from Charon.OpenMode import OpenMode  #T o open local files with the selected open mode.
+# The supported file types.
+from Charon.filetypes.OpenPackagingConvention import OpenPackagingConvention
+from Charon.filetypes.UltimakerFormatPackage import UltimakerFormatPackage
+from Charon.filetypes.GCodeFile import GCodeFile
+from Charon.filetypes.GCodeGzFile import GCodeGzFile
 
-from .FileInterface import FileInterface #The interface we're implementing.
-from .OpenMode import OpenMode #To open local files with the selected open mode.
-
-#The supported file types.
-from .filetypes.UltimakerFormatPackage import UltimakerFormatPackage
-from .filetypes.GCodeFile import GCodeFile
-from .filetypes.GCodeGzFile import GCodeGzFile
 
 extension_to_mime = {
+    ".opc": "application/x-opc",
```

Comment:
I'm not even sure that there is such a thing as a .opc file. I've not been able to find any reference to it what so ever. So i think we should just keep the OPC support as a sort of interface class (as a raw OPC file doesn't make any sense. It needs to have an actual implementation.)